training/week1/miscellaneous/dig-into-the-blood/README.md

```diff
@@ -11,7 +11,11 @@ The image is in here: [Dig+Into+The+Blood.png](https://challenges.reply.com/tamt
 
 Use an _Image Editor_ (I used [Paint.NET](https://www.getpaint.net/)) to alter the image colors and then the **FLAG** will appear:
 
+<<<<<<< HEAD:week1/Dig_into_the_Blood/README.md
+```verbatim
+=======
 ```text
+>>>>>>> 00907800a5560f16e8a54984f036e419c9bde9b9:training/week1/miscellaneous/dig-into-the-blood/README.md
```

Comment:
Remove all those changes, the file is already OK in the main branch

Reply:
Done